### PR TITLE
add more resources to training resources page

### PR DESCRIPTION
**Pull request by davidwyld**
Adding some more resources to the page, including:
- linkedin learning courses
- BES guides
- RIT courses

### CODE REVIEW

File: pages/training/programming/index.md [hunk 1]
```diff
@@ -66,8 +66,39 @@ There are Python IDEs with data science focus such as [Rodeo](https://rodeo.yhat
 
 Here’s a demonstration of a notebook which is used to illustrate working through a machine learning exercise:  [https://nbviewer.jupyter.org/github/jdwittenauer/ipython-notebooks/blob/master/notebooks/ml/ML-Exercise1.ipynb](https://nbviewer.jupyter.org/github/jdwittenauer/ipython-notebooks/blob/master/notebooks/ml/ML-Exercise1.ipynb)
 
+
+### IT Services Training Courses
```
Comment (willfurnass) [inline pages/training/programming/index.md:70]: Could reference back to this from HPC section further down the page?

File: pages/training/programming/index.md [hunk 2]
```diff
@@ -66,8 +66,39 @@ There are Python IDEs with data science focus such as [Rodeo](https://rodeo.yhat
...
+  * [R](https://www.linkedin.com/learning/code-clinic-r-2)
+  * [Python](https://www.linkedin.com/learning/code-clinic-python-2)
+  * [C++](https://www.linkedin.com/learning/code-clinic-c-plus-plus-2)
+  * [Java](https://www.linkedin.com/learning/code-clinic-java-2)
+
+However, there are great courses on almost any topic in research computing. Why not try one of these?
+  * [Docker for developers](https://www.linkedin.com/learning/docker-for-developers-2/practical-backend-with-compose)
+  * [Unit Testing and Test Driven Development in Python](https://www.linkedin.com/learning/unit-testing-and-test-driven-development-in-python)
+  * [Learning the R Tidyverse](https://www.linkedin.com/learning/learning-the-r-tidyverse)
+  * [Learning MATLAB](https://www.linkedin.com/learning/learning-matlab-2)
```
Comment (willfurnass) [inline pages/training/programming/index.md:93]: Could add MATLAB Academy (see RSES Feb newsletter) elsewhere on this page.

**Comment by davidwyld** [comment]
Thanks @willfurnass !

**Review by drj11 — APPROVED** [review]
Looks, good, i particularly like the inclusion of the BES guide.

**Comment by davidwyld** [comment]
> Looks, good, i particularly like the inclusion of the BES guide.

Love those guides.

**Pull request merged**